$NetBSD: patch-ai,v 1.1 2003/01/05 03:31:25 dsainty Exp $

jinclude.h is not part of the libjpeg interface, and isn't available from the
pkgsrc libjpeg installation.  But we need to include <stdio.h> ourselves for
the benefit of "jpeglib.h".

--- SrcShared/EmJPEG.h.orig	Sat Mar 30 01:11:14 2002
+++ SrcShared/EmJPEG.h	Wed Jan  1 18:05:36 2003
@@ -30,7 +30,7 @@
 
 extern "C"
 {
-	#include "jinclude.h"
+	#include <stdio.h>
 	#include "jpeglib.h"
 	#include "jerror.h"
 }
